app/scripts/metamask-controller.js

```diff
@@ -3221,6 +3222,16 @@ export default class MetamaskController extends EventEmitter {
     };
   }
 
+  async getTokenSymbol(address) {
+    try {
+      const details =
+        await this.assetsContractController.getTokenStandardAndDetails(address);
```

Comment:
[this method does a bunch of extra network calls](https://github.com/MetaMask/core/blob/69e95b1b9b3409aff65d1723552768e78166fc2c/packages/assets-controllers/src/AssetsContractController.ts#L300) in excess of what we need. We should expose the getERC20TokenSymbol method on the `AssetsContractController` and call that instead.

Reply:
hey @adonesky1 ,
thnx for your comment, this will be done in another task